wg-olm/README.md

```diff
@@ -0,0 +1,27 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# OLM Working Group
+
+To organize community input for the Operator Lifecycle Manager project and it's integration with other Operator Framework projects.
```

Comment:
Same nit as above

Reply:
Resolved in the latest version of the PR.